CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# v1.4.0-M3 (Jun 2020)
+  * Tune GC to improve execution time performance ([`KT-19076`](https://youtrack.jetbrains.com/issue/KT-19076)).
+  * Support @Throws annotation ([`KT-34848`](https://youtrack.jetbrains.com/issue/KT-34848))
```

Comment:
K/N has already supported `@Throws` annotation. KT-34848 adds `annotation class Throws` to common. I'm not sure this belongs to Native changelog.

Reply:
Ok, removed.